src/basic/reference-helpers.js

```diff
@@ -1,11 +1,25 @@
+import R from 'ramda';
 import Utils from '../utils/utils';
+import TangramReference from '../utils/reference';
 
-var ReferenceHelpers = {};
 
-let RH = ReferenceHelpers;
+/*
+  INTERNAL REFERENCE FUNCTIONS
+ */
 
-export default ReferenceHelpers;
+const curryComp = Utils.curryCompose3;
 
+
+/*
+  REFERENCE HELPER
+ */
+var ReferenceHelper = {};
+
+let RH = ReferenceHelper;
```

Comment:
I personally don't like names like this. `ReferenceHelper` is longer but it's clear what you are using all over the code